pages/base_page.py

```diff
@@ -173,10 +171,6 @@ def report_list(self):
             select = Select(report_dropdown)
             return [opt.text for opt in select.options]
 
-        @property
-        def is_exploitable_crash_report_present(self):
-            return self.is_element_present(*self._exploitable_crash_report_locator)
-
```

Comment:
I understand why you removed this, but is there a use case for privileged users to see that the exploitable crash report is present? The new test only confirms it isn't there for non-privileged users.